src/js/utils/DSLUtil.js

```diff
@@ -194,6 +194,19 @@ const DSLUtil = {
           memo[prop] = matchingNodes
             .map((ast) => ast.filterParams.text)
             .join(' ');
+
+          // Also append whatever whitespace remains at the end of the raw value
+          //
+          // NOTE: This is a trick to allow dynamic updates of a React component
+          //       since white spaces at the end of the expression are not part
+          //       of a regular DSL expression and are trimmed.
+          //
+          const whitespaceRegex = /\s+$/;
+          const tailingWaitspace = whitespaceRegex.exec(expression.value);
```

Comment:
I think you actually mean `trailingWhiteSpace`
![](http://www.ronstoyshop.com/category_images/trailer_catalog/enclosed_cargo_trailers/5_wide_trailers/trailers_car_mate_enclosed_cargo_trailer_500_cm508cc.jpg)
![](http://24.media.tumblr.com/tumblr_m3x648wxbj1ru99qvo1_500.png)
![](https://media.giphy.com/media/U1cPyeDtP2i64/giphy.gif)